packages/editor/src/functions/addMediaNode.ts

```diff
@@ -169,10 +194,28 @@ export async function addMediaNode(
           ...extraComponentJson
         ],
         parent!,
-        before,
-        name
+        before
       )
 
+      const reactor = startReactor(() => {
+        const entity = UUIDComponent.useEntityByUUID(entityUUID)
+        const modelComponent = useOptionalComponent(entity, ModelComponent)
+
+        useImmediateEffect(() => {
+          if (!modelComponent) return
+          const pathArray = url.split('/')
+          const lastIndex = pathArray.length - 1
+          const fileNameWithExt = pathArray[lastIndex]
+          const fileNameArray = fileNameWithExt.split('.')
+          const name = decodeURI(fileNameArray[0])
+          setComponent(entity, NameComponent, name)
+
+          reactor.stop()
+        }, [modelComponent])
+
+        return null
+      })
```

Comment:
let's remove this from this PR, not in scope

Reply:
okay i will do this, there is a fix for load lookdev asset with multiple lookdev components, i will leave tthis in this pr